### DISCUSSION TITLE
2024 Conferences

### DISCUSSION THREAD

**Discussion by arthurrge** [aiaa-deic/computation-ai#13] [General]
We had proposed finding time at 2023's AVIATION conference - I'm not sure if that happened or not.

Suggest using this discussion group to point out conferences and coordinate connecting as helpful.

**Comment by arthurrge** [comment]
Coming up related to this group is [SC|23](https://sc23.supercomputing.org/). I will be there, curious who else plans to attend / potentially connect there.

**Reply by coleyc** [reply]
I am currently planning to attend SC 23 and would like to meet up with anyone else who will be there!

**Reply by coleyc** [reply]
@arthurrge not too many other responses on here but let me know your thoughts on meeting up at SC 23

**Comment by opinonfischer** [comment]
I live 25 minutes from downtown so not registered but won't be far :)

**Comment by arthurrge** [comment]
SC|23 was (as always) a wall to wall ravenous consumer of attention, and unfortunately did not afford any side-meetings.

SciTech is coming upon us quickly following the end of year, and I plan to be there Sunday to Thursday.

My sense is this will be an opportunity to have discussion/debate that has been a challenge virtually, perhaps following the DEIC general session/s. I'm sure, like me, we're all trying to wind down to EOY. I'd like to park here the suggestion for that, as well as continue to encourage contribution through the github site.

Thank you and I wish all the best for the holidays and year's end.

**Comment by arthurrge** [comment]
I stumbled on this conference - here are my condensed notes / references:

### AI4SE & SE4AI Workshop 2023 
Sep 27-28, Systems Engineering Research Center (SERC) <[event page](https://sercuarc.org/event/ai4se-se4ai-workshop-2023/)>

**KEYNOTES**

**AI@Speed & Scale (Army Digital Transformation)**
Ms. Jennifer Swanson, Deputy Assistant Secretary of the Army for Data, Engineering & Software, ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/01.Keynote%20Speaker%20-%20Jennifer%20Swanson.pdf)  )
Office of the Assistant Secretary of the Army (Acquisition, Logistics and Technology)
- Project Linchpin - DoD Trusted AI environment
- Process for model deployment into operations
- AI Risk Framework (draft/in progress)

**Secure, Robust, & Scalable AI/Autonomy**
Dr. Kimberly Sablon, Principal Director, Trusted Artificial Intelligence and Autonomy, ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/03.Keynote%20Speaker%20-%20Kimberly%20Sablon.pdf) )
Office of the Under Secretary of Defense for Research and Engineering

**DEVCOM ARMAMENTS CENTER PERSPECTIVES**
Dr. Myron Hohil, SSTM for Artificial Intelligence, Sensors & Collaborative Munitions, ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/02.US%20Army%20DEVCOM%20Armaments%20Center%20Perspectives.pdf) )
US Army DEVCOM Armaments Center

**Lockheed Martin Transformation - You might not recognize us**
Mr. Michael “Rabbi” Harasimowicz, Director, AI Innovations, Lockheed Martin ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Keynote%20Michael%20%C3%A2%C2%80%C2%9CRabbi%C3%A2%C2%80%C2%9D%20Harasimowicz%20-%20Transforming%20Lockheed_%20Committed%20to%20Change.pdf)  )

**SERC + INCOSE PERSPECTIVES: Roadmap for AI and SE**
Dr. Peter Beling, Director, Intelligent Systems Division, Virginia Tech National Security Institute and SERC Research Council Member; Dr. Ali Raz, Assistant Professor, George Mason University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/SERC%20Perspective%20and%20INCOSE%20AIWG%20-%20Beiling%20-%20Raz.pdf)  )

- Interesting paths, progression levels mapped
- [INCOSE AI Working Group](https://www.incose.org/communities/working-groups-initiatives/artificial-intelligence-systems) tutorials offered 

**PANELS**

**PANEL: CERTIFICATION OF LEARNING-BASED SYSTEMS**
Moderator: Dr. Zoe Szajnfarber, The George Washington University
Dr. Jonathan Barkand, Defense Acquisition University
Dr. Jason Summers, ARiA

**CREDENTIALING MACHINE LEARNING**
Dr. Tyler Cody, Virginia Tech National Security Institute ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Panel1_01.%20Tyler%20Codypptx.pdf)  )

**CERTIFY / QUALIFY AI - INSIGHTS FROM AVIATION**
MAJ Christine Krueger, The George Washington University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Panel1_02.%20Christine%20Krueger.pdf) )

**PANEL: APPLICATION OF LLMs to SYSTEMS ENGINEERING**
Moderator: Dr. Myron Hohil, US Army DEVCOM Armaments Center

**ENHANCING SE WITH AI**
Mr. Amir Abrari, SPEC Innovations ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Panel%202_01.%20Amir%20Abrari.pdf)  )
Artificial Intelligence (AI) Large Language Models (LLMs) for Systems Engineering (SE) Tasks
Mr. John Crissman, CNA Corporation ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Panel%202_02.%20John%20Crissman.pdf)  )
The Use of Large Language Models LLM /"Generative" Systems in Secure and Restricted Settings
Dr. Carlo Lipizzi, Stevens Institute of Technology ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Panel%202_03.%20Carlo%20Lipizzi.pdf) )
(This provides a nice overview/contextualization of LLMs)

**PANEL: COMPARING ETHICAL AND SAFETY FRAMEWORKS FOR PROMOTING TRUST IN AI SYSTEMS**
Moderator: Mr. Andy Lacher, NASA Langley Research Center
Dr. David Broniatowski, The George Washington University 
Mr. Chuck Howell, MITRE
Dr. Kim Wasson, Joby Aviation

**TRACKS**
**Trustworthy AI Track**
**Establishing a Measurement Science for Joint Work with Models**
Dr. Peter Denno, National Institute of Standards and Technology ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_01.%20Peter%20Denno.pdf) )
Application of Systems Theoretic Process Analysis (STPA) to the Mission Assurance of AI-Enabled Systems in Aerospace
Mr. Robert Crombie, Mr. Daniel Byrne, The Aerospace Corporation ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_02.%20Robert%20Crombie.pdf))
[!] Application of the Traditional Materiel Release Process to the DoD Ethical Principles of Artificial Intelligence, and Roadmap
Mr. Benjamin Werner, Mr. Benjamin Schumeg, US Army DEVCOM Armaments Center ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_03.%20Benjamin%20Werner.pdf) )
- Design for Reliability for AI framework
- STTR-A22B-T002 (Metrics and Methods for Verification, Validation, Assurance and Trust of Machine Learning Models & Data for Safety-Critical Applications in Armaments Systems)
- DOD Ethical Principles of AI (with detail): Responsible, Equitable, Traceable, Reliable, Governable

**Using Artificial Intelligence to Accelerate Deployment of Learning-Based Systems**
Dr. Jason Gregory, US Army DEVCOM Research Laboratory; Dr. Satyandra K. Gupta, University of Southern California ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_04.%20Jason%20Gregory.pdf)  )
- Accelerate testing & deployment (AI-enabled experimental design)
- Challenges: Complexity + Context

**Conformal Prediction for Testing and Evaluation of Intelligent/ML Systems**
Mr. Paul Horton, Georgia Tech Research Institute ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_05.%20Paul%20Horton.pdf) )
Evaluating the Explainability and Interpretability of AI Systems
Ms. Alaysha Shearn ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%201%20Trustworthy%20AI_06.%20Alaysha%20Shearn.pdf)  )
Dr. David Broniatowski, Dr. Armin Vosoughi, The George Washington University, Dr. Valerie Reyna, Cornell University 

**AI4SE Track**
**AI-Assisted Generation of SysML Models to Accelerate Agile Digital Engineering**
Mr. Matthew Jones, Ms. Erin Crabb, Leidos ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%202%20AI4SE_01.%20Matthew%20Jones.pdf) )
Requirement Discovery Using Embedded Knowledge Graph with ChatGPT
Dr. Nipa Phojanamongkolkij, Mr. Braxton VanGundy, Mr. Ramana Polavarapu, Mr. Joshua Bonner, NASA Langley Research Center; Dr. Barclay Brown, Collins Aerospace ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%202%20AI4SE_02.%20Nipa%20Phojanamongkolkij%20-%20Braxton%20VanGundy.pdf) )
**LLM Based SysML Virtual Assistant**
Mr. Alexander Demagall, Mr. Gabe Apanza, Dr. Daniel Selva, Texas A&M University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%202%20AI4SE_03.%20Alexander%20Demagall.pdf)  )
Melding Minds and Machines: Harnessing AI Capabilities to Further Model-Based Guided Engineering
Ms. Melinda Ong, Mr. Andrew Gaskins, Booz Allen Hamilton ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%202%20AI4SE_04.%20Melinda%20Ong.pdf)  )
- MBGE = model-based guided engineering (model generation, descriptive generation, analysis, validation)
- GSE = Generative Systems Engineering
- AI-driven requirements generation

**A Constructed System of Analysis to Enable Automation and Reasoning in Multi-Model Analysis**
Mr. Daniel Dunbar, Dr. Mark Blackburn, Dr. Tom Hagedorn, Stevens Institute of Technology ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%202%20AI4SE_05.%20Daniel%20Dunbar.pdf) )
- Digital Thread = connected (multi-model / multi-simulation) analyses
- Digital Engineering Framework for Integration & Interoperability
- Ontology-aligned Data
- System of Analyses
- MISD = Model Interface Specification Diagram
- SSVL = Semantic System Verification Layer

**Large Language Model Enabled Generation of Systems Engineering Artifacts**
Dr. Paul Wach, Dr. Taylan Topcu, Mr. Mohammed Husain, Virginia Tech ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track+2+AI4SE_06.+Paul+Wach+%2B+Mohammed+Husain.pdf)  )
Human AI/Teaming Track
Is the Machine a Partner or a Tool? A Major Issue of Human-AI Teaming
Dr. Guy Boy, Paris Saclay University (CentraleSupélec) & ESTIA Institute of Technology ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_01.%20Guy%20Boy.pdf) )
- White paper: [Epistemological Approach to Human Systems Interaction](https://www.sciencedirect.com/science/article/pii/S0160791X23001033) (HSI) 
- Suggests, akin to Technical Readiness Levels - HRL & ORL
- Human Readiness Level & Organization Readiness Level

**What is Human-in-the-Loop, Really?**
Mr. Aditya Singh, Dr. Zoe Szajnfarber, The George Washington University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_02.%20Aditya%20Singh.pdf) )
- Interesting framework / taxonomy suggested
- (AI employing) Human-on-the-Loop
- (Hybrid) Human-in-the-Loop
- (Human) AI-under-the-Loop

**Managing the Continuity of Human Biases in AI SE Applications**
Dr. Steven Conrad, Colorado State University; Ms. Heidi Perry, MIT Lincoln Laboratory ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_03.%20Steve%20Conrad.pdf) )
- Human Augmentation vs. Human + Technology Teaming vs. Assurance

**Opportunities and Risks of Incorporating LLMs in the Systems Engineering and Design Workflow: A Case Study of Robotic System Design Process**
Dr. Zoe Szajnfarber*, Mr. Oladele (Richard) Adeyeye, Dr. Robert Pless, The George Washington University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_04.%20Zoe%20Szajnfarber.pdf))
The Need to Update Design and Certification Performance Standards for Operator Intervention of Autonomous Systems
Dr. Lance Sherry, Dr. George Donohue, Ms. Jomana Bashata, George Mason University ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_05.%20Lance%20Sherry.pdf) )
Mission Engineering in Healthcare and What We Can Apply to the Military
Mr. Richard Threlkeld, Dr. Cihan Dagli, Missouri University of Science and Technology ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%203_06.%20Richard%20Threlkeld.pdf)  )

**SE4AI Track
Hiring Trained Animals: Generative AI Patterns and Practices for Systems Engineering**
Dr. Barclay Brown, Collins Aerospace ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%204_01.%20Barclay%20Brown.pdf) )
Systems’ Perspectives for AI Acceleration in Large Enterprises and Government Agencies
Dr. Jyotirmay Gadewadikar, Mr. Jim Lockett, Mr. Jeremy Marshall, MITRE ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track+4_02.+Jyotirmay+Gadewadikar+%2B+Jim+Lockett.pdf) )
Test and Evaluation (T&E) of Complex AI-Enabled Systems (AIES)
Dr. Flo Reeder*, Ms. Elena Charnetzki, MITRE ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%204_03.%20Flo%20Reeder.pdf) )
Systems Engineering Processes to Test AI Right (SEPTAR)
Ms. Carol Pomales, Dr. Flo Reeder, Dr. Carlos Balhana, Mr. Jim Lockett, Ms. Ivy Chen, MITRE ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%204_04.%20Carol%20Pomales.pdf) )
A Systems Engineering Perspective on Safety of AI-based Systems
Mr. Reginald Holmes, Dr. Hanumanthrao “Rao” Kannan, University of Alabama in Huntsville ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%204_05.%20Reginald%20Holmes.pdf) )
- R3 re: Safety (Robustness, Reliability, Resilience)

**Simulating the Tradespace with Synthetic Data**
Dr. Cameron Turner, Clemson University; Dr. Matthew Castanier, Dr. David Gorsich, Dr. Greg Hartman, US Army Ground Vehicle Systems Center ([Link](https://sercproddata.s3.us-east-2.amazonaws.com/publication_documents/reports/Track%204_06.%20Cameron%20Turner.pdf)  )

**Comment by arthurrge** [comment]
SciTech 2025 season has begun!

https://www.aiaa.org/SciTech/call-for-content

**Comment by arthurrge** [comment]
This [USAF Modeling & Simulation Summit](https://www.dafmss.org/conference-agenda) just came to our awareness. Is anyone engaged with it? Awkwardly overlaps the [Dayton Digital Transformation Summit](https://sinclairwfd.regfox.com/2024-dayton-digital-transformation-summit), also heavily DAF-oriented.
